src/lib/button/button.scss

```diff
@@ -79,6 +80,10 @@
   border-radius: inherit;
   pointer-events: none;
   opacity: 0;
+
+  @include md-high-contrast {
+    background-color: rgba(white, 0.5);
```

Comment:
Done, although I had to flip the conditions since the colors are inverted. Might have to re-test this after we have dark themes implemented.